xesmf/frontend.py

```diff
@@ -6,10 +6,46 @@
 import xarray as xr
 import os
 
-from . backend import (esmf_grid, add_corner,
-                       esmf_regrid_build, esmf_regrid_finalize)
-
-from . smm import read_weights, apply_weights
+from .backend import (esmf_grid, add_corner, esmf_regrid_build,
+                      esmf_regrid_finalize)
+
+from .smm import read_weights, apply_weights
+
+
+def get_latlon_name(ds, boundary=False):
+    if boundary:
+        try:
+            # COARDS netCDF complaint
+            if 'lat_b' in ds.variables:
+                lat_name = 'lat_b'
+                lon_name = 'lon_b'
+            # NETCDF CF 1.6 complaint
+            elif 'latitude_b' in ds.variables:
```

Comment:
Does the COARD convention specify boundary names? I can't find anything [here](https://ferret.pmel.noaa.gov/Ferret/documentation/coards-netcdf-conventions)

CF convention uses 'lat_bnds' for [boundaries](http://www.cgd.ucar.edu/cms/eaton/netcdf/CF-20010629.htm#bnds) but its size is (N, 2), not N+1 as expected by ESMF (https://github.com/JiaweiZhuang/xESMF/issues/32#issuecomment-419496538). 

The current 'lat_b' (including the proposed 'latitude_b') does not follow existing convention, but is rather my arbitrary choice. I don't particularly like this choice but switching to 'lat_bnds' is even more confusing due to the above size issue. So we can't really say we are "looking for CF-compliant names" here.

Also a typo: "complaint" -> "compliant"

Reply:
lat_bnds is for regional boundary conditions. It isn't for the same thing as edge or corners of the grid cell.